srcIdl/perftest.idl

```diff
@@ -18,6 +22,11 @@ struct TestData_t {
     unsigned long timestamp_usec;
     long latency_ping;
     sequence<octet, MAX_BOUNDED_SEQ_SIZE> bin_data;
+  #ifdef RTI_CUSTOM_TYPE
+    long custom_type_size;
```

Comment:
it is the serialize size of the sample that we send on the wire (less the overhead)